pandas/tests/frame/test_sorting.py

```diff
@@ -89,6 +89,22 @@ def test_sort_values(self):
         with tm.assert_raises_regex(ValueError, msg):
             frame.sort_values(by=['A', 'B'], axis=0, ascending=[True] * 5)
 
+        # GH 16836
```

Comment:
The test is actually superfluous. The fix to `test_sort_nat_values_in_int_columns` is sufficient to cover the issue. Do you still want to keep it?